core/chains/evm/gas/models.go

```diff
@@ -352,11 +365,11 @@ func BumpDynamicFeeOnly(config Config, lggr logger.SugaredLogger, currentTipCap
 // the Tip only. Unfortunately due to a flaw of how EIP-1559 is implemented we
 // have to bump FeeCap by at least 10% each time we bump the tip cap.
 // See: https://github.com/ethereum/go-ethereum/issues/24284
-func bumpDynamicFee(cfg Config, lggr logger.SugaredLogger, currentTipCap, currentBaseFee *big.Int, originalFee DynamicFee, maxGasPriceWei *big.Int) (bumpedFee DynamicFee, err error) {
+func bumpDynamicFee(cfg Config, lggr logger.SugaredLogger, currentTipCap, currentBaseFee *utils.Wei, originalFee DynamicFee, maxGasPriceWei *utils.Wei) (bumpedFee DynamicFee, err error) {
 	maxGasPrice := getMaxGasPrice(maxGasPriceWei, cfg)
 	baselineTipCap := bigmath.Max(originalFee.TipCap, cfg.EvmGasTipCapDefault())
 
-	bumpedTipCap := increaseByPercentageOrIncrement(baselineTipCap, cfg.EvmGasBumpPercent(), cfg.EvmGasBumpWei())
+	bumpedTipCap := utils.NewWei(baselineTipCap).IncreaseByPercentageOrIncrement(uint32(cfg.EvmGasBumpPercent()), cfg.EvmGasBumpWei())
```

Comment:
WDYT about making this logic a little more direct?
```suggestion
	bumpedTipCap := utils.MaxWei(baselineTipCap.AddPercentage(uint32(cfg.EvmGasBumpPercent()), baselineTiCap.Add(cfg.EvmGasBumpWei()))
```

Reply:
To my eye this is less easy to understand